projects/core/src/lib/layers/legend/legend-raster/raster-legend.component.ts

```diff
@@ -56,6 +56,52 @@ export class CastMeasurementToContinuousPipe implements PipeTransform {
     }
 }
 
+export function unifyDecimals(values: number[]): number[] {
+    // Early return if all values differ by more than 1
+    if (values.every((x) => x >= 1)) {
```

Comment:
I guess instead of first checking that everything is >= 1, just do the one loop and check that the values are apart.

Reply:
Right, that check was completely unnecessary and a remnant from when I used to only cut off all the decimals when the integer part was at least three digits. I removed the if-check.